interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/customize-appearance/report-style-sheets.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Report Style Sheets
+owner: Mary Sammal
+---
+# Report Style Sheets
+
+You can combine [report styles](customize-appearance\report-visual-styles.md) into a style sheet and reuse them in reports. This topic explains how to create and use style sheets in reports.
+
+## Save Styles as Style Sheets
+
+Press the caption button on the toolbar's Styles group to invoke the Styles Editor.
```

Comment:
Why 'Styles' editor? Style editor?

Reply:
Because this editor allows you to customize a collection of styles

Reply:
Yes, but surely it is a 'style' editor, even if you can edit 100s of styles?